docs/api/python/cluster.rst

```diff
@@ -321,3 +321,49 @@ Runhouse offers two options for enabling TLS/SSL on a cluster with Caddy:
    self-signed certificates to use for the cluster. These certs will not be verified by a CA.
 2. *Using Caddy to generate CA verified certs*: Provide the :code:`domain` argument. Caddy will then obtain
    certificates from Let's Encrypt on-demand when a client connects for the first time.
+
+
```

Comment:
did you add this in the `notebooks` repo? we technically don't modify rsts directly, and modify `ipynb` and then sync